_config.yml

```diff
@@ -33,43 +33,6 @@ exclude:
   - .jekyll-cache/
 languages: ["en", "es", "fr", "it", "pt", "ru"]
 exclude_from_localizations: ["assets"]
-chapters:
```

Comment:
why delete this? iirc this is what results in an empty chapters dropdown menu

if we want to remove that for some reason, i think we should just remove it entirely and add the chapters link to the hamburger menu

Reply:
Imo, also for tracking purposes, would be better if people opened/went to the chapters page itself. I think adding this to top of "site.links" would be good (the main hamburger menu), and remove/hide the social media links in the top hamburger menu, while leaving the footer the same